.github/actions/install-pkg/action.yml

```diff
@@ -0,0 +1,35 @@
+name: Install earthaccess Python package
+description: Install earthaccess Python package and testing dependencies
+
+inputs:
+  python-version:
+    description: Version of Python to use
+    required: true
+  cache-key:
+    description: Cache key to use; combined with python-version.
+    default: ci
+    required: false
```

Comment:
Looks like this input isn't used. Is it worth keeping?

Reply:
Double doh. :rofl:

Reply:
1cf3750